### PR TITLE
Update stoa0169a.stoa002.opp-lat2.xml

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updated the header to current OGL standards.

### CODE REVIEW

File: data/stoa0169a/stoa002/stoa0169a.stoa002.opp-lat2.xml [hunk 1]
```diff
@@ -2,7 +2,9 @@
 <teiHeader>
     <fileDesc>
             <titleStmt>
-                <title>Corpus Scriptorum Ecclesiasticorum Latinorum (CSEL)</title>
+                <title>Evangeliorum libri quattuor</title>
+                <author>Juvencus, Caius Vettius Aquilinus<author>
```
Comment (lcerrato) [inline data/stoa0169a/stoa002/stoa0169a.stoa002.opp-lat2.xml:6]: The author tag is not closed.

File: data/stoa0169a/stoa002/stoa0169a.stoa002.opp-lat2.xml [hunk 2]
```diff
@@ -2,7 +2,9 @@
 <teiHeader>
     <fileDesc>
             <titleStmt>
-                <title>Corpus Scriptorum Ecclesiasticorum Latinorum (CSEL)</title>
+                <title>Evangeliorum libri quattuor</title>
```
Comment (lcerrato) [inline data/stoa0169a/stoa002/stoa0169a.stoa002.opp-lat2.xml:5]: Lang attribute needed.

File: data/stoa0169a/stoa002/stoa0169a.stoa002.opp-lat2.xml (hunk 1 above)
Comment (lcerrato) [reply data/stoa0169a/stoa002/stoa0169a.stoa002.opp-lat2.xml:6]: Lang attribute needed.

File: data/stoa0169a/stoa002/stoa0169a.stoa002.opp-lat2.xml [hunk 3]
```diff
@@ -42,18 +44,19 @@
             <sourceDesc>
                 <listBibl xml:lang="lat">
                     <biblStruct>
-                        <monogr>
-                            <editor>
+                        <monogr>                                                       
+                         <author>Juvencus, Caius Vettius Aquilinus</author>
```
Comment (lcerrato) [inline data/stoa0169a/stoa002/stoa0169a.stoa002.opp-lat2.xml:48]: Author and title here both need a Lang attribute

File: data/stoa0169a/stoa002/stoa0169a.stoa002.opp-lat2.xml [hunk 4]
```diff
@@ -42,18 +44,19 @@
...
+                                    <addName xml:lang="deu">Johann Huemer</addName>
                                 </persName>
-                            </editor> 
-                          <author ref="http://data.perseus.org/catalog/urn:cite:perseus:cattg.1542.1">Iuvencius</author>  
-                          <title ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0169a.stoa002.opp-lat2">Evangeliorum Libri Quattuor</title>
+                            </editor>
                             <imprint>
-                                <publisher>Österreichische Akademie der Wissenschaften (ÖAW)</publisher>
-                                <pubPlace>Wien, Österreich</pubPlace>
+                                <title>Corpus scriptorum ecclesiasticorum Latinorum</title>
```
Comment (lcerrato) [inline data/stoa0169a/stoa002/stoa0169a.stoa002.opp-lat2.xml:57]: Title needs Lang attribute

**Review by lcerrato — CHANGES_REQUESTED** [review]

**Comment by lcerrato** [comment]
@AlisonBabeu 
The check is failing because the author tag isn’t closed.

File: data/stoa0169a/stoa002/stoa0169a.stoa002.opp-lat2.xml (hunk 4 above)
Comment (AlisonBabeu) [reply data/stoa0169a/stoa002/stoa0169a.stoa002.opp-lat2.xml]: Hi @lcerrato I think I made all the necessary corrections. Thank you for reviewing.

**Comment by lcerrato** [comment]
The `<title xml:lang="lat">Corpus scriptorum ecclesiasticorum Latinorum</title>` in the `<imprint>` level is not allowed. I don't know where we want to include the series. It looks like it can go just before the ref target.
`<series xml:lang="lat">Corpus scriptorum ecclesiasticorum Latinorum</series>`

**Review by lcerrato — APPROVED** [review]

**Pull request merged**